test/performance/load-test/load-test.yaml

```diff
@@ -62,25 +62,30 @@ spec:
             image: knative.dev/serving/test/performance/load-test
             args:
             - "-flavor=zero"
+            - "-github-token=/var/secret/github-token"
```

Comment:
These should just be built-in

Reply:
Do you also suggest moving them to `config-mako`?

Reply:
No, I think the paths should be baked in.  Things like the channel and whatnot should move to config-mako.